insights/parsers/installed_rpms.py

```diff
@@ -226,13 +226,6 @@ def get_min(self, package_name):
         else:
             return min(self.packages[package_name])
 
-    @property
-    def is_hypervisor(self):
-        """bool: True if ".el[6|7]ev" exists in "vdsm".release, else False."""
-        rpm = self.get_max("vdsm")
-        return (True if rpm and rpm.release.endswith((".el6ev", ".el7ev")) else
-                False)
-
     # re-export get_max/min with more descriptive names
```

Comment:
@psachin this method should be marked as deprecated instead of being removed, see [Feature Deprecation] (https://insights-core.readthedocs.io/en/latest/api.html#feature-deprecation) for more information.  Please include information on which method its replaced by.

Reply:
@bfahr The method which replaces `is_hypervisor` is internal to insights-plugins repo, users not having access to internal rules will not be able to use that method. Besides all existing plugins using `is_hypervisor` are now using that (internal)method, hence I decided to remove the function instead of deprecating it.

Please suggest how we should move ahead.

Reply:
@psachin the normal process is to mark it as deprecated because insights-core is a public project and we don't know about all uses of the the code so we deprecate and then at a later time we'll remove the code.

Reply:
@bfahr 

Thanks for the feedback. The method is now deprecated.

Reply:
Hey @psachin, since the deprecated function is still in the code, we probably should keep the test code.  Can you add those tests back?

Reply:
@miclark Sure.

Reply:
@miclark @bfahr 

I used `warnings.warn()` to deprecate `is_hypervisor` because the tests failed with `RecursionError` due to recursive nature of `@property`. Let me know if that's ok or should I raise an issue to come up with an approach to safely deprecate the property methods in the framework.